### ISSUE TITLE
complext table cell got AVT violation that no header defined

### ISSUE THREAD

**Issue by lj7720** [IBM/carbon-components-angular#1745]
## Detailed description

> Describe in detail the issue you're having.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

Table

> What did you expect to happen? What happened instead? What would you like to see changed?

No violation for Accessibility checker 

> What browser are you working in?

Firefox 85.0.2

> What version of the Carbon Design System are you using?

3.54.2

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

ADDI

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/74904663/109271433-7ae9fc80-784a-11eb-9ef6-d6a41580c002.png)
![image](https://user-images.githubusercontent.com/74904663/109272358-ba651880-784b-11eb-9010-a5499c87897a.png)
![image](https://user-images.githubusercontent.com/74904663/109272418-cf41ac00-784b-11eb-925d-390bcbd91b8b.png)

* Notes

**Comment by lj7720** [comment]
hi, @cal-smith This issue was still pending and it will affect our next accessibility compliance report, would you please arrange to resolve asap? Thanks.
@stanislavgeorgiev

**Comment by lj7720** [comment]
@Donisius After upgrade to latest carbon release (4.52.1), the violation still exists, would you please help for a check? Thanks.
![image](https://user-images.githubusercontent.com/74904663/121167161-41cb1a80-c884-11eb-8c32-da2dee4d15fc.png)

@ninja511

**Comment by Donisius** [comment]
@lj7720 sorry I took a while to get back to you, are you setting the headers through the table model or are you creating a custom header like in [this example](https://github.com/IBM/carbon-components-angular/blob/master/src/table/table.stories.ts#L503-L533)? If you are setting them similar to that example you need to set the `scope="col"` property on your `th`s.

**Comment by lj7720** [comment]
@Donisius Thanks for your comments. Now we can resolve it by adding the property in your sample. It's closed then.
@ninja511
